mcfix_base/data/res_partner_demo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
remove encoding

Reply:
Are you sure? I thought utf-8 was meant to be replaced only in py files.